data/categories.js

```diff
@@ -0,0 +1,15 @@
+// An enum for categories of professions
+
+class Category {
```

Comment:
Enums need to be used by the Routes as well for parsing the data from string to Enum.
So, I guess it's better to move these Enum files into separate directory "enums/" or "constants/" etc
What do you think?